automations/cube.py

```diff
@@ -0,0 +1,92 @@
+from components.lifter import Lifter
+from components.intake import Intake
+from magicbot import StateMachine, state, timed_state
+
+
+class CubeManager(StateMachine):
+    lifter: Lifter
+    intake: Intake
+
+    @timed_state(first=True, must_finish=True, duration=0.5, next_state="grabbing_cube")
+    def initialize_auto_cube(self):
```

Comment:
I think you're forgetting to call this.

Reply:
See autonomous/autonomous.py line 102

Reply:
Oh right, derp. I think you forgot to do it for the simple drive forward auto though.